.github/workflows/ci.yml

```diff
@@ -124,14 +123,14 @@ jobs:
         if: ${{ matrix.android }}
         run: |
           cd android-project && \
-          ./gradlew assemble${{ matrix.debug == 'true' && 'Debug' || 'Release' }} \
+          ./gradlew assembleRelease \
```

Comment:
I doubt `assembleRelease` will work on ci as it needs private keys. Until we figure this out, let's do `assembleDebug`?